2018/proposal.md

```diff
@@ -0,0 +1,90 @@
+PyCon AU 2018 Education Seminar Proposal
+========================================
+
+If accepted, the PyCon Australia Education Seminar will bring together educators who are using Python
+in teaching or looking for new ways to use coding in education. The goal is to share stories, strategies
+and techniques for engaging students and teaching with Python that can be directly applied by other educators. 
+
+Python has long had reputation as an effective beginner programming language. Bringing together educators and
+developers helps developers provide better support for beginners in Python and its libraries, as well as
+giving educators the rare opportunity to develop their own technical skills further.
+
+As in previous years, talk topics at the miniconf will be open to any educational use of Python but the core
+focus will continue to be on primary and secondary school education. This includes both formal classroom
+instruction and extracurricular activities.
+
+However, some changes and additions are also being proposed based on our experiences in previous iterations
+of the event.
+
+Event Organisers
+----------------
+
+* Lead organiser: Nick Coghlan
+  * 2015 Education Seminar founder & coordinator, 2016/17 co-coordinator with Katie Bell
+  * CPython core developer
+  * Former member of the Python Software Foundation Board of Directors (2014-16)
+
+* Co-organiser: Nicky Ringland? (Bruce Fuda?, Amanda Hogan?)
+  * Australian Computing Academy: https://aca.edu.au/about.html
+
+Event Outline
+-------------
+
+Prior to the afternoon tea break, the 2018 Education Seminar will follow the same
+structure as in previous years: an invited speaker prior to morning tea, 3 speakers
+from the Call for Proposals prior to lunch, then 3 more speakers prior to afternoon
+tea.
+
+For the final session, we'd like to work with the main conference organisers to
+arrange to run a "Student Showcase", inspired by the well-received student
+presentations at PyCon Australia 2017.
+
+The general idea of this part of the proposal would be to:
+
+* Provide a way for teachers & senior secondary students to nominate as potential
+  presenters at the Showcase (similar to the way Gungahlin College students did
+  through the regular CfP for 2017)
+* Accept up to 6 projects for presentation (these would each be given a 15 minute
+  presentation slot *without* any question & answer time, and a ~5 minute break
+  in between slots. Moving back and forth between the Student Showcase and the other
+  specialist tracks would generally be discouraged)
+* Work with the conference organisers to provide a way for teachers, family members,
+  and students to attend just the showcase session for a relatively nominal cost
+  (e.g. $15)
```

Comment:
This bit is a bit problematic I think. Teachers and students aren't going to like this. I don't think we'll be inundated with an influx of parents and friends coming along to watch the kids. It's a Thursday, after all.
From talking to schools, unfortunately $15 or $10 isn't necessarily a nominal cost. :( 

I'd be happier trying to get a sponsor to chip in for the cost of the kids' attendance for the showcase session if we need something there. I'm confident I can convince a company to chip in little bit to cover some extra afternoon tea for the kids, if that's what we need.

Reply:
Specialist tracks will be on the Friday.

I was thinking regardless of whether these were made generally available or not, accepted students should get a handful of zero-cost tickets to give out (e.g. themselves plus up to 5 others), so I'll reword this to make it clearer we have a lot of flexibility in choosing an approach we think will work well for teachers, students, families, friends, and the conference itself.

Reply:
I've switched the example out with one where each selected student would be given half-a-dozen Showcase-only tickets to distribute.

Reply:
True - sorry - Friday. Brain snap there. But still hard for students' friends/family to get time off school/work.

Yup, that sgtm. I don't even think they'll need that many, but I guess it doesn't hurt!